lib/pure/net.nim

```diff
@@ -1313,8 +1343,7 @@ proc select(readfd: Socket, timeout = 500): int =
   if readfd.hasDataBuffered:
     return 1
 
-  var fds = @[readfd.fd]
-  result = selectRead(fds, timeout)
+  result = timeoutRead(readfd.fd, timeout)
```

Comment:
We shouldn't switch a proc called "select" to use "poll" under the hood. That's very misleading.

Reply:
I renamed it to `timeoutRead`.